src/App/Pages/Vault/ViewPageViewModel.cs

```diff
@@ -741,16 +743,38 @@ public bool ShowHiddenValue
                 });
         }
 
+        public string ValueText
+        {
+            get
+            {
+                if (IsBooleanType)
+                {
+                    return _field.Value == "true" ? "" : "";
+                }
+                else if (IsLinkedType)
+                {
+                    var i18nKey = _cipher.LinkedFieldI18nKey(Field.LinkedId.GetValueOrDefault());
+                    return " " + _i18nService.T(i18nKey);
```

Comment:
Special chars should be formatted as unicode values (ex: "\uf24a")

Reply:
We can probably skip this for now (I forgot @Hinton already has a full conversion (to unicode values in a constants class) in motion with the icon updates.

Reply:
I'll follow @mportune-bw 's advice and leave it for now